auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java

```diff
@@ -65,25 +66,47 @@ private void createAuditEvent(RequestDetails theRequestDetails, boolean wasSucce
     auditEvent.setType(new Coding("Query", "110112", "Audit event: Query has been made"));
     auditEvent.setRecorded(new Date());
 
+    //retrieve initially created plandefinition by using empty search parameter map
+    IBundleProvider allPlanDefinitions = myPlanDefinitionDao.search(new SearchParameterMap());
+    PlanDefinition initiallyCreatedPlanDefinition;
+    if (allPlanDefinitions != null && !allPlanDefinitions.isEmpty() && allPlanDefinitions.size() > 0) {
+      initiallyCreatedPlanDefinition = (PlanDefinition) allPlanDefinitions.getResources(0, 1).get(0);
+    } else {
+      throw new ExceptionInInitializerError("Apparently no plandefinition has been created during initialization");
+    }
+
+    //add based on extension
+    Extension basedOnExtension = new Extension();
+    basedOnExtension.setUrl("http://aist.fh-hagenberg.at/fhir/extensions/auditevent-basedon-extension");
```

Comment:
Is this similar to a xml namespace where just any url can occur or should this be our server url?

Reply:
yes, it is like a xml namespace that we are using here.